app/(admin)/admin/notifications/page.tsx

```diff
@@ -42,30 +47,87 @@ const AdminNotifications = (): JSX.Element => {
     await sendEmailNotifications({ content, groupUsers, subject });
   };
 
+  /**
+   * Function to handle radio selection logic.
+   * @param value - Value of the radio buttons.
+   */
+  const handleRadioChange = (value: string): void => {
+    if (value === 'all') {
+      getLeagueData();
+    }
+  };
+
   return (
-    <section data-testid="admin-notifications-content">
-      <Button
-        label="Email Testers"
-        type="button"
-        onClick={getLeagueData}
-        data-testid="email-testers"
-      />
-      <form onSubmit={handleSubmit}>
-        <Label htmlFor="subject">Subject:</Label>
-        <Input
-          type="text"
-          id="subject"
-          data-testid="subject-text"
-          onChange={(e) => setSubject(e.target.value)}
-        />
-        <Label htmlFor="content">Content:</Label>
-        <textarea
-          name="content"
-          id="content"
-          onChange={(e) => setContent(e.target.value)}
-          data-testid="content-text"
+    <section
+      className="flex flex-col space-y-6"
+      data-testid="admin-notifications-content"
+    >
+      <p>
+        Choose the users you would like to email in INSERT_CURRENT_LEAGUE_HERE
+      </p>
+      <RadioGroupDefault
+        defaultValue=""
+        onValueChange={handleRadioChange}
+        required
```

Comment:
nit: This is outside of scope, but something to consider for a future ticket is to add logic to `sendEmailNotifications` in case we have users changing this attribute in the browser console

Reply:
I agree. We ought to add more robust checks for the form and inputs.